apps/communications/contacts/js/activities_vcard.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+/* global LazyLoader, VCardReader, Contacts */
+/* exported VcardActivityHandler */
+
+var VcardActivityHandler = function() {
```

Comment:
please use a conventional module pattern here and do not abuse from arrow function declaration.